README.md

```diff
@@ -1,2 +1,49 @@
-# Main-2022-09-22
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse Project
+
+> For this project, I had to create "Hello world" repository and a feature branch that i cloned to my local machine, I then added index.HTML file and a Styles.css File, I set up linters for HTML&CSS and then created a .gitignore file to make sure certain files do not get published when i commit my changes from my local repo to my GitHub Master Repo, The goal here is to master all of the tools and best practices learned about in previous steps of The Program, which I will be using in all Microverse projects and most likely in my future job as well, so it is important to know the processes required!
+
+
+## Built With
+
+- HTML & CSS
+- Github Workflow Actions, HTML&CSS Linters
+- Virtual Studio Code, GitHub
+
+
+
+## Getting Started
+
+**This Project was to make sure i understand the basic tools and methods i will be using on a daily basis during the course of my microverse programm as well as future work life.**
+
+
+
+## Authors
+
+👤 **Jerome**
+
+- GitHub: [@githubhandle](https://github.com/187jjay187)
+- Twitter: [@twitterhandle](https://twitter.com/187jjay187)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jerome-osman-137605a4)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
[OPTIONAL] This should contain the link to the issues page of your repository. Kindly update it to make your project look more proffessional

Reply:
Thank you for the insight into these issues, i really do appreciate it, i have resolved it